token-swap/js/package.json

```diff
@@ -15,21 +15,31 @@
   "publishConfig": {
     "access": "public"
   },
+  "browser": {
+    "./lib/index.cjs.js": "./lib/index.browser.esm.js",
+    "./lib/index.esm.js": "./lib/index.browser.esm.js"
+  },
   "main": "lib/index.cjs.js",
   "module": "lib/index.esm.js",
   "types": "lib/index.d.ts",
+  "browserslist": [
+    "defaults",
+    "not IE 11",
+    "maintained node versions"
+  ],
   "files": [
     "/lib",
-    "/module.flow.js"
+    "/module.flow.js",
```

Comment:
Can you add `module.flow.js` to the local .gitignore?

Reply:
Done